contrib/docker/entrypoint/entrypoint.go

```diff
@@ -70,8 +77,7 @@ func main() {
 		decredData := os.Getenv("DECRED_DATA")
```

Comment:
The two lines to declare `decredData` and `dcrdAppData` are duplicated and can be moved up out of the switch